projects/js-packages/publicize-components/src/components/form/index.js

```diff
@@ -72,7 +72,12 @@ export default function PublicizeForm( {
 	const hasInstagramConnection = connections.some(
 		connection => connection.service_name === 'instagram-business'
 	);
-
+	const { showShareLimits, numberOfSharesRemaining } = useSelect( select => {
+		return {
+			showShareLimits: select( socialStore ).showShareLimits(),
+			numberOfSharesRemaining: select( socialStore ).numberOfSharesRemaining(),
+		};
+	} );
```

Comment:
I've just noticed this could be:

```suggestion
	const { showShareLimits, numberOfSharesRemaining } = useSelect( select => ( {
			showShareLimits: select( socialStore ).showShareLimits(),
			numberOfSharesRemaining: select( socialStore ).numberOfSharesRemaining(),
	} ) );
```

Another stylistic change though. Take it or leave it. We can apply a lint rule if we decide we prefer one over the other.